packages/desktop-client/src/components/sidebar/PrimaryButtons.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-strict-ignore
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what problem do we have here that this is added?
```suggestion
```

Reply:
Removed